Detectors/TPC/workflow/src/TPCTimeSeriesSpec.cxx

```diff
@@ -1040,6 +1040,7 @@ class TPCTimeSeries : public Task
           const auto& trkOrig = tracksTPC[iTrk];
           const bool isNearestVtx = (idxITSTPC.back() == -1); // is nearest vertex in case no vertex was found
           const float mx_ITS = hasITSTPC ? tracksITSTPC[idxITSTPC.front()].getX() : -1;
+          const float pt_ITS = hasITSTPC ? tracksITSTPC[idxITSTPC.front()].getPt() : -1;
```

Comment:
@matthias-kleiner, @miranov25  passed by my office to make sure this PR is served and told that he wanted q/pT rather than pT (also for the TPC?). Also, it seems he wanted the matching chi2 (`tracksITSTPC[idxITSTPC.front()].getChi2Match()`) rather than ITS ...

Reply:
Thanks, for the TPC we already have the q/pt